doc/Contribute.md

```diff
@@ -100,6 +101,27 @@ associated documentation, if we have any.
 |__top__/rules                                                          |Supporting bazel definitions.                          |
 |[__top__/testing](tests.md)                                            |Bazel targets to run CF smoke and acceptance tests.    |
 
+
+## Updating subcharts
+
+The kubecf helm chart includes a number of subcharts. They are declared in
+[requirements.yaml](../deploy/helm/kubecf/requirements.yaml). For the
+convenience of development they are included in unpacked form directly in
+this repo, so version changes can be inspected with regular `git` tools,
+and the subcharts can be searched with `grep` etc.
+
+The procedure to update the version of a subchart is:
+
+```
+cd deploy/helm/kubecf
+vi requirements.yaml
+rm -rf charts/*
+helm dep up
+for CHART in charts/*tgz; do tar xfz $CHART -C charts; rm $CHART; done
```

Comment:
Of course my first thought is "what if the file name has a space in it".  `C:\Program Files` has traumatized me forever.

Reply:
I doubt this will happen, but agree that the code should be robust enough to handle that, so will make a change.

Reply:
Fixed in a262c5e